src/fhir/PlanDefinition_opc.json

```diff
@@ -257,11 +257,9 @@
       "status": "active",
       "kind": "CommunicationRequest",
       "timingTiming": {
-        "schedule": {
-          "event": [
-            "2023-09-25T15:33:00.000Z"
-          ]
-        }
+        "event": [
```

Comment:
@achen2401  your change here is fine w/ me. I thought I was being consistent with what we'd done previously, but I now think that I wasn't. The spec is slightly ambiguous from my pov; there are only a few references at chat.fhir.org but Bryn Rhodes does have this example, which matches what we've been doing: https://chat.fhir.org/#narrow/stream/179166-implementers/topic/Immunization.20Schedule/near/204020735

Reply:
thanks @mcjustin I see.. the frontend code is looking for `event: []` syntax, i.e. https://github.com/Ahryman40k/typescript-fhir-types/blob/4.0.0/src/R4/Resource/RTTI_Timing.ts#L52 . It complained when I tried to change it to `scheduled: event[]` It might be that they changed the syntax for R4.